packages/appstore/src/components/modals/account-type-modal/account-type-modal.tsx

```diff
@@ -115,7 +115,8 @@ const MT5AccountTypeModal = () => {
             case localize('Derived'):
                 setAccountType({ category: 'real', type: 'synthetic' });
                 break;
-            case localize('Swap-Free') || localize('Swap Free'):
+            case localize('Swap-Free'):
+            case localize('Swap Free'):
```

Comment:
Why same condition is used twice?

Reply:
because it other languages (i.e. spanish, Indonesian) there is no dash (-) and it end up at default again